src/pages/signup/Signup.js

```diff
@@ -0,0 +1,356 @@
+import React, { Component } from "react";
+import Collapse from '../../component/Collapse';
+import Dropdown from '../../component/Dropdown';
+import styled from 'styled-components';
+import './Signup.css';
+
+const SWrapper = styled.div`
+    display: block;
+    text-align: center;
+    font-weight: bold;
+`
+const SHeader = styled.div`
+    margin-top:10px;
+    margin-bottom:10px;
+    font-size: 40px;
+    color: #a417ff;
+`
+const STitle = styled.div`
+    margin-top:10px;
+    margin-bottom:-10px;
+    color: #283140;
+    font-size: 22px;
+    text-align: center;
+`
+const SForm = styled.form`
+    display: inline-block; 
+    font-size: 14px;
+    text-align: left;
+    line-height: 1.9;
+    width: 420px;
+`
+const SHr = styled.hr`
+    border: 0;
+    height: 0;
+    border-top: 0.6px solid #c6c6c6;
+    border-bottom: 0.6px solid #c6c6c6;
+`
+const SGroup = styled.div`
+    position:relative;
+`
+const SInput = styled.input`
+    width: 370px;
+    height: 28px;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 8px 8px 8px 35px;
+    border: none;
+    border-bottom: 1.4px solid #c6c6c6;
+    color: #283140;
+`
+const SButton = styled.button`
+    width: 170px;
+    height: 50px;
+    font-size: 20px;
+    font-weight: bold;
+    color: white;
+    background-color: #a417ff;
+`
+const SButton2 = styled.button`
+    width: 120px;
+    height: 40px;
+    margin-left: 10px;
+    font-size: 14px;
+    font-weight: bold;
+    color: white;
+    background-color: #c6c6c6;
+`
+const SLabel = styled.label`
+    position:absolute;
+    left:10px;
+    top: 10px;
+`
+const SSubmit = styled.div`
+    text-align:center;
+`
+const SNote = styled.p`
+    font-size: 11px;
+    color: #c6c6c6;
+    margin-top: 2px;
+    margin-left: 10px;
+`
+const SAgree = styled.div`
+    margin-top: 20px;
+    margin-bottom: 30px;
+    width: 415px;
+`
+const SPurpleLabel = styled.label`
+    color: #a417ff;
+`
+const SAgreeLabel = styled.label`
+    color: #c6c6c6;
+`
+const SCheck= styled.input`
+    margin-left: 10px;
+`
+
+class SignUp extends Component {
+
+    constructor(){
+        super()
+        this.state = {
+            school: '',
```

Comment:
흠 이건 객체 하나로 묵어도 좋을 것 같은데!
```suggestion
            loginDTO { school: '',
            email:',
            }
```
밑에도 쭈루룩 해서 이런식으로?